components/chat/MermaidDiagram.tsx

```
@@ -0,0 +1,492 @@
+              </>
+            ) : (
+              <>
+                <LucideCopy className="size-4"/> <span className="ml-1">Copy</span>
+              </>
+            )}
+          </button>
+        </div>
+      </div>
+      <div
+        className="p-4 bg-white rounded-b-lg overflow-x-auto mermaid-diagram"
+        dangerouslySetInnerHTML={{ __html: svg }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)